site/content/internal/onboarding/manager-guide.md

```diff
@@ -32,4 +32,10 @@ weight: 30
 - Add new staff to the group
 
 ### Core Committer Mug
-- Message @lindy and ask her to send the new hire their core committer mug
+- Message @lindy.isherwood and ask her to send the new hire their core committer mug
+
+### Release Dates Google Calendar
+- Message @lindy.isherwood and ask her to add the new hire to the Mattermost Release Dates google calendar
+
+### Jira Access
+- Message @amy.blais and ask her to add the new hire to ``jira-developers`` and ``internal`` groups in Jira
```

Comment:
I'm assuming we'll automate this at some point?

Reply:
That's a good point, I'll add a note for myself as next steps.